src/components/Process/Header.tsx

```diff
@@ -87,9 +88,11 @@ const ProcessHeader = () => {
                 <ElectionSchedule textAlign='left' color='process.info_title' format='PPPp' />
               </Flex>
             </Flex>
-            <Box display={{ base: 'none', xl: 'flex' }}>
-              <ShareModalButton caption={t('share.election_share_text')} text={t('share.election_share_btn_text')} />
-            </Box>
+            {import.meta.env.CLIENT !== 'pxll' && (
+              <Box display={{ base: 'none', xl: 'flex' }}>
+                <ShareModalButton caption={t('share.election_share_text')} text={t('share.election_share_btn_text')} />
+              </Box>
+            )}
```

Comment:
As I said in other cases, since this is a specific client modification, simply remove the block rather than adding a conditional.